sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -241,6 +241,7 @@ export async function saveFakeApplication(
     program?: EducationProgram;
     offering?: EducationProgramOffering;
     programYear?: ProgramYear;
+    msfaaNumber?: MSFAANumber;
```

Comment:
Is `msfaaNumber` referenced anywhere in the method?

Reply:
Updated